test/e2e/app-dir/ppr-partial-hydration/ppr-partial-hydration.test.ts

```diff
@@ -60,28 +66,35 @@ describe('PPR - partial hydration', () => {
             await browser.elementByCssInstant('#dynamic-fallback').text()
           ).toContain('Loading...')
         },
-        1000,
-        50
+        // This can take a while? It's unclear why... The delay appears pretty
+        // random.
+        /* duration */ 15_000 // ms
```

Comment:
I'm really not sure why this timeout needs to be so long. It seems very inconsistent how long this takes.